css/style.css

```diff
@@ -0,0 +1,1138 @@
+    margin-left: 0;
+    display: grid;
+    grid-template-rows: 1fr 108px;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .aside-info {
+    margin: -2px 34px 0 -16px;
+  }
+
+  .container__boxes {
+    margin: -12px 0 0 66px;
+  }
+
+  .container__box {
+    margin-bottom: 21px;
+  }
+
+  .box-about {
+    display: grid;
+    grid-column: 1/3;
+  }
+
+  .box-about__container {
+    justify-content: center;
+    margin: 0 64px 40px 0;
+  }
+
+  .container__aside-info {
+    margin-top: 24px;
+    margin-bottom: 58px;
+  }
+
+  .box-about__img-ellipse {
+    width: 31.18px;
+  }
+
+  .box-about__learn {
+    font-size: 16px;
+  }
+
+  .box-about__button {
+    width: 220.84px;
+    height: 55.21px;
+    margin: 0 0 12px 44px;
+  }
+
+  .footer {
+    flex-direction: column;
+  }
+
+  .footer__section {
+    order: 1;
+    margin-bottom: 28px;
+  }
+
+  .footer__aside {
+    order: 2;
+    max-width: 1300px;
+  }
+
+  .footer__section-title {
+    font-size: 148px;
+    text-align: center;
+    margin: 38px 0 0 75px;
+  }
+
+  .section__img-car {
+    display: none;
+  }
+
+  .section__img-car-ipad {
+    display: inline-flex;
+    width: 95%;
+    margin-top: -135px;
+  }
+
+  .footer-img {
+    display: none;
+  }
+
+  .footer__aside {
+    display: grid;
+    min-width: 0;
+    grid-template-rows: 1fr 108px;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .aside__safe {
+    display: none;
+  }
+
+  .box-about__footer-one {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    margin-right: -18px;
+  }
+
+  .footer__dev {
+    display: none;
+  }
+
+  .aside__about {
+    margin: 14px 14px 0 -20px;
+  }
+
+  .wrapper {
+    margin: -13px 0 32px 41px;
+  }
+
+  .wrapper__box {
+    padding-right: 0;
+    margin-right: -11px;
+  }
+}
+
+@media (max-width: 480px) {
+  .container__title {
+    display: none;
+  }
+
+  .container__img {
+    display: none;
+  }
+
+  .container__img-phone {
+    display: inline-flex;
+  }
+
+  .aside-info {
+    margin: -34px -17px 0 -17px;
+  }
+
+  .container__boxes {
+    display: none;
+  }
+
+  .box-about {
+    display: none;
+  }
+
+  .container__aside {
+    display: flex;
+    min-width: 0;
+    margin: -11px 0 0 -7px;
+  }
+
+  .container__aside-info {
+    margin-top: -14px;
+  }
+
+  .footer__section-title {
+    font-size: 91px;
+    width: 100%;
+    margin: 9px 0 -82px -8px;
+  }
+
+  .section__img-car-ipad {
+    margin: -65px 0 53px 5px;
+  }
+
+  .wrapper {
+    display: none;
+  }
+
+  .aside__safe {
+    display: none;
+  }
+
+  .footer__aside {
+    display: flex;
+    min-width: 0;
+    margin: -50px 0 0 -17px;
+  }
+
+  .footer__aside-title {
+    margin: 16px -56px 0 11px;
+  }
+
+  .footer__aside-info {
+    margin: 10px -51px 0 11px;
+  }
+
+  .aside__about {
+    margin-top: -32px;
+  }
+}
+
+@media (max-width: 550px) {
+  .main__container {
+    flex-direction: column;
+    padding: 0 36px;
+    justify-content: center;
+    margin-top: 264px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)